### DISCUSSION TITLE
How to use pretrain model in Unet encoder?

### DISCUSSION THREAD

**Discussion by ramchandracheke** [Project-MONAI/tutorials#1223] [Q&A] [answered]
Hi,
I am training a segmentation model on the pancreas dataset using the standard Unet model. I am using pre-trained weights for the encoder. However, I have observed that pre-trained weights are changed by training and a randomly initialised model's performance is similar to the pre-trained model. Can you please suggest what can be done in this case.
Thanks,

**Comment by KumoLiu** [comment, marked as answer]
Hi @ramchandracheke, retrain a model with pre-trained indeed changed the weight when training since it needs to be updated. And also there is possible the performance of a randomly initialized model is similar to the pre-trained model, which may indicate that the pre-trained weight is not helpful for training.
You could also try to freeze the weight of the encode several epochs after you load the pre-trained weight.

Hope it can help you. Thanks!

**Reply by ramchandracheke** [reply]
Thanks for your input. I will update you as I get the results.
